### PR TITLE
Dl breakpoint changer

### PR DESCRIPTION
**Pull request by rmoylan**
#2 -> https://github.com/IBM-Design/design-language-website/issues/2

This creates components - InputRange, StickyContainer, TypesetExample, necessary for use in BreakpointChanger also created.

### CODE REVIEW

File: packages/addons-website/scss/components/_website-side-nav.scss [hunk 1]
```diff
@@ -319,7 +319,7 @@ a.#{$prefix}--side-nav__link > .#{$prefix}--side-nav__icon {
   opacity: 0;
   will-change: opacity;
   transition: $transition--expansion opacity $carbon--standard-easing,
-  $transition--expansion visibility $carbon--standard-easing;
+    $transition--expansion visibility $carbon--standard-easing;
```
Comment (jnm2377) [inline packages/addons-website/scss/components/_website-side-nav.scss]: example of the transition tokens I was talking about earlier.

**Comment by jeanservaas** [comment]
Hey guys, not sure if it's ready for design review yet, but if this is the breakpoint changer for our expressive type tab I just want to make sure design gets to review it too! thanks!

**Comment by alisonjoseph** [comment]
@jeanservaas https://deploy-preview-1271--carbon-website.netlify.com/storybook/ to review

**Comment by alisonjoseph** [comment]
This is looking great, we need to add padding-right 25% to the example blocks, and I believe styles should match this page for font sizes and spacing for the  example specs https://next.carbondesignsystem.com/guidelines/typography/productive

Would also be great if you could update the page above and also expressive with the new component. (and we can remove the current TypeSpec component)

**Comment by rmoylan** [comment]
@alisonjoseph I addressed your comments, but I'm waiting for one piece of art direction before able to check.

**Comment by alisonjoseph** [comment]
should be able to use `$focus` for the focus color

**Pull request merged**